### PR TITLE
Tied map/model update on selection of filter menu

### PR DESCRIPTION
**Pull request by isuftin**

### CODE REVIEW

File: src/main/webapp/js/views/ModelMapView.js [hunk 1]
```diff
@@ -56,59 +58,60 @@ define([
...
 					log.debug('Got model layers');
 					getModelLayerNamesPromise.resolve(response[0]);
 					$('#map-loading-div').hide();
 				},
-				error : function(jqxhr, textStatus) {
+				error: function (jqxhr, textStatus) {
 					log.debug('Error in retrieving model layers');
 					getModelLayerNamesPromise.reject(textStatus);
 					$('#map-loading-div').hide();
+					// TODO - Let's not do alerts. Need something less abrasive
```
Comment (mbucknell) [inline src/main/webapp/js/views/ModelMapView.js]: Agreed. We could do something like an AlertView which maybe shows a modal. Did something like that in GDP.

File: src/main/webapp/js/views/ModelMapView.js [hunk 2]
```diff
@@ -56,59 +58,60 @@ define([
...
-					params : {
-						LAYERS : response.FlowLayerName,
-						STYLES : response.FlowLayerDefaultStyleName
+					serverType: 'geoserver',
+					params: {
+						LAYERS: response.FlowLayerName,
+						STYLES: response.FlowLayerDefaultStyleName
 					},
-					url : response.EndpointUrl
+					url: response.EndpointUrl
```
Comment (mbucknell) [inline src/main/webapp/js/views/ModelMapView.js]: Is this really a style thing? I actually like a space after the property.

Comment (isuftin) [inline src/main/webapp/js/views/ModelMapView.js]: It's just how Netbeans autoformats. JSHint wasn't complaining or anything.

File: src/main/webapp/js/views/ModelMapView.js (hunk 1 above)
Comment (isuftin) [inline src/main/webapp/js/views/ModelMapView.js]: In CCH we use [Alertify](http://fabien-d.github.io/alertify.js/) as something that isn't overbearing. 

That project does seem to not be maintained anymore. That's a recent thing though.

**Pull request merged**